news/_posts/core-weekly/2021-11-08-coreweekly-44-2021.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 44 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-11-08
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 1st to Sunday 7th of November 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+[Hacktoberfest](https://hacktoberfest.digitalocean.com/) is over. But the PrestaShop project continues, stronger than ever.
+
+In case you did not see it, [Autoupgrade module 4.3.0](https://github.com/PrestaShop/autoupgrade/releases/tag/v4.13.0) was released last week!
+
+Also last week, the project’s tenth monthly public demo session was held and streamed live on [Wednesday 3rd of November 2021](https://build.prestashop.com/news/public-oss-demo-10-2021/). If you did not watch it and you want to know more about the future of the PrestaShop _open source project_ (and not the software), check what @ttoine presented during the 10th PrestaShop Public Demo!
```

Comment:
```suggestion
Also last week, the project’s tenth monthly public demo session was held and streamed live on [Wednesday 3rd of November 2021](https://build.prestashop.com/news/public-oss-demo-10-2021/). If you did not watch it and you want to know more about the future of the PrestaShop _open source project_ (and not the software), check what [@ttoine](https://github.com/ttoine) presented during the 10th PrestaShop Public Demo!
```

Reply:
Not needed, GitHub automatically generates the links ❤️ 

See https://github.com/PrestaShop/prestashop.github.io/edit/master/news/_posts/public-oss-demo/2021-11-04-public-oss-demo-10-2021.md and https://build.prestashop.com/news/public-oss-demo-10-2021/